OpenOversight/app/main/views.py

```diff
@@ -380,14 +381,15 @@ def redirect_add_assignment(officer_id: int):
 def add_assignment(officer_id: int):
     form = AssignmentForm()
     officer = db.session.get(Officer, officer_id)
+    if not officer:
+        flash("Officer not found")
+        abort(HTTPStatus.NOT_FOUND)
```

Comment:
Similar logic to: https://github.com/lucyparsons/OpenOversight/pull/1124/files#r1779369649